### DISCUSSION TITLE
Disable - usePreferredTextZoom Android - Capacitor without PhoneGap Plugin

### DISCUSSION THREAD

**Discussion by dchedufa** [ionic-team/capacitor#3335] [Q&A] [answered]
Hi, All! Im very new to the community, im struggling to get fix font size on Android devices. The issue is that if the user selects text scaling as part of the accessibility settings it will generate the text sizes increased.

I have seen some responses that indicate using PhoneGap accessibility plugin, but this was a very old solution and overkilling to add the plugin just for this case.

Suggestion...
    window.MobileAccessibility.usePreferredTextZoom(false)

Other suggestions were made for Cordova.
IOS build is working without any specific setup.

Thanks in advanced!

**Comment by imhoffd** [comment, marked as answer]
First of all, welcome! 

There's a feature request for this functionality [here](https://github.com/ionic-team/capacitor-plugins/issues/3). You can follow that issue for updates.

**Comment by dchedufa** [comment]
Thanks @dwieeb hope i can contribute to this community!

**Comment by dchedufa** [comment]
Just for adding the outcome of what I did to have the fix in the meantime. 

Within the Bridge class, there is a initWebView function were you can set the zoom by adding the  following lines:
`  private void initWebView() {
    WebSettings settings = webView.getSettings();
    settings.setTextZoom(100);
    settings.setSupportZoom(false);`

**Comment by Cyral** [comment]
Adding on to what @dchedufa said: You can call this from MainActivity without editing the capacitor source like so: (Call this method after this.init(...) in MainActivity)

```
  public void adjustFontScale() {
    // https://github.com/ionic-team/capacitor/discussions/3335
    WebSettings settings = bridge.getWebView().getSettings();
    settings.setTextZoom(100);
    settings.setSupportZoom(false);
  }
```

**Reply by mabdelfattah** [reply]
@Cyral I'm really sorry, but I have no experience with native Android development. So, if I am using Capacitor, which file should I edit to use your lines?

**Reply by Cyral** [reply]
@mabdelfattah in MainActivity.java in the Android portion of your app. Not sure on the exact path off the top of my head.

**Reply by mabdelfattah** [reply]
I just found Capacitor already implemented it in their core. But thank you so much for responding.

**Reply by FabrizioAndre** [reply]
@mabdelfattah Hello, and with which plugin did you do it? I have seen that it can be @capacitor/text-zoom, but I still cannot solve it
